### DISCUSSION TITLE
Can't get touch to work (works when calibrating though)

### DISCUSSION THREAD

**Discussion by DreamT80** [HASwitchPlate/openHASP#608] [General]
Hi,

I've been running openHASP for a while now, using it with the openHASP integration in HA. The hardware is a D1 Mini (ESP32-WROOM-32) and a Lolin TFT 2.4" Touch Shield.

I'm supplying the JSONL pages file to the plate via the HA integration (i have a basic pages.jsonl stored on the plate aswell, just in case of the connection being down). I have setup one page (p1) where i display different sensor values that I update dynamically via HA. Works great.

Now, I want to have multiple pages, and I want to be able to toggle between these pages with buttons a la [this example](https://www.openhasp.com/0.7.0/examples/pagination/), but it won't work. Nothing happens when I press a button.

I thought that it first was a hardware issue, but the page calibration works, so I don't think it's the problem.

Does anyone have have any pointers on how to debug this? To I have to enable a setting or something to make it work?

**Comment by spetryjohnson** [comment]
I have a different device (the Sunton cheap yellow display), but the very first time I flashed it I couldn't get the touchscreen to work either.

On a second try I followed the docs a little more closely and I did a full flash of the nightly binary using Tasmota PyFlasher, then I installed the OTA binary from the most recent Github Action w/ an appropriate asset, and then I ran calibration.

After that I configured my plate through HA and touch was working fine.

I have no idea if any of those steps are relevant or not, so YMMV...
